### PR TITLE
Update Event Message EIP

### PR DESCRIPTION
**Pull request by poorna2152**
## Purpose
$subject

### CODE REVIEW

**Comment by poorna2152** [comment]
The image here needs to be updated. Created an issue for this https://github.com/wso2/docs-mi/issues/773

File: en/docs/learn/enterprise-integration-patterns/message-construction/event-message.md [hunk 1]
```diff
@@ -13,89 +13,167 @@ The Event Message EIP is used for reliable, asynchronous event notification betw
...
+       </target>
+    </proxy>
+    ```
+=== "EventObserver1"
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <proxy name="EventObserver1" transports="jms" startOnLoad="true" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+       <target>
+           <inSequence>
+              <property name="OUT_ONLY" value="true"/>
```
Comment (SanojPunchihewa) [inline en/docs/learn/enterprise-integration-patterns/message-construction/event-message.md:55]: We don't need the `OUT_ONLY` property here as are not forwarding the message. Shall we test it and update this.

File: en/docs/learn/enterprise-integration-patterns/message-construction/event-message.md [hunk 2]
```diff
@@ -13,89 +13,167 @@ The Event Message EIP is used for reliable, asynchronous event notification betw
...
+    [[transport.jms.listener]]
+    name = "myTopicConnectionFactory"
+    parameter.initial_naming_factory = "org.apache.activemq.jndi.ActiveMQInitialContextFactory"
+    parameter.provider_url = "tcp://localhost:61616"
+    parameter.connection_factory_name = "TopicConnectionFactory"
+    parameter.connection_factory_type = "topic"
 
-   3. Enter the name `stockquote` for the topic and then click **Add Topic**.
+    [[transport.jms.sender]]
+    name = "myTopicSender"
```
Comment (SanojPunchihewa) [inline en/docs/learn/enterprise-integration-patterns/message-construction/event-message.md:131]: We have missed using this in the JMS sender. Please update it accordinly

File: en/docs/learn/enterprise-integration-patterns/message-construction/event-message.md [hunk 3]
```diff
@@ -13,89 +13,167 @@ The Event Message EIP is used for reliable, asynchronous event notification betw
...
 </soapenv:Envelope>
 ```
 
-Observe the two Axis2 server instances. Both instances will receive the request, which was sent from the client.
-
-Output of both Axis Server consoles:
+## Analyze the output
 
-samples.services.SimpleStockQuoteService :: Generating quote for : foo
+When you execute the command above, the request is sent to the StockQuoteProxy. Notice the following processed server log in WSO2 MI output:
```
Comment (SanojPunchihewa) [inline en/docs/learn/enterprise-integration-patterns/message-construction/event-message.md:162]: We have not defined a `StockQuoteProxy` in the example. Are you referring to the JMS topic here?

Comment (poorna2152) [reply en/docs/learn/enterprise-integration-patterns/message-construction/event-message.md]: Updated

**Review by SanojPunchihewa — APPROVED** [review]

**Pull request merged**